js/src/features/human-resouce/components/EmployeeLocationEditForm.tsx

```diff
@@ -23,6 +23,8 @@ interface Props {
   edit: (data: EditEmployeeLocationData) => void;
 }
 
+const DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
Try to use formatISO function. And i suggest you to use [date-fns fp branch](https://github.com/date-fns/date-fns/blob/master/docs/fp.md).

Reply:
fix